frontend/package.json

```diff
@@ -135,7 +132,6 @@
     "jest": "^26.6.3",
     "jest-fail-on-console": "^3.1.1",
     "jest-transform-stub": "^2.0.0",
-    "module-alias": "^2.2.2",
```

Comment:
See the note on `playwright.config.ts` for why this is removed (no longer necessary)